recipes/hangar/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "hangar" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 85bbb1e8fe0cac56ca86257283a96472bd8410443d1b819d0e6b67f0a0d63214
+
+build:
+  number: 1
+  skip: true  # [py<36]
+  entry_points:
+    - hangar = hangar.cli:main
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+
+requirements:
+  host:
+    - blosc-hdf5-plugin  # [not win]
+    - click
```

Comment:
You probably don't need almost all of these in `host`? Most of the time just having `python` and `pip` is enough.

Reply:
fixed!